src/Symfony/Component/Console/Helper/TableCell.php

```diff
@@ -45,6 +45,14 @@ public function __construct($value = '', array $options = array())
         }
 
         $this->options = array_merge($this->options, $options);
+
+        if (0 === (int) $this->options['colspan']) {
```

Comment:
Why `>=` ? the colspan can be any positive integer,
Do you mean `if (0 <= (int) $this->options['colspan'])` ?